crates/rsonpath-lib/src/query.rs

```diff
@@ -34,45 +34,13 @@ pub mod automaton;
 pub mod builder;
 pub mod error;
 mod label;
+mod nonnegative_array_index;
 mod parser;
 pub use label::Label;
-
 use log::*;
+pub use nonnegative_array_index::NonNegativeArrayIndex;
 use std::fmt::{self, Display};
 
```

Comment:
Sorry for nitpicking, but we should have the order consistent. I need to get this fixed on the formatter level (#135), but for now please reorder these manually:

- all `mod` directives
- all `pub use` directives
- blank line
- all `use` directives

Similarly to how `engine.rs` is structured.

Reply:
Reordered per-se.